src/EnergyPlus/PlantCondLoopOperation.cc

```diff
@@ -1539,104 +1563,107 @@ void FindCompSPInput(EnergyPlusData &state,
+                                    ErrorsFound = true;
+                                }
+                            }
+                        }
+                        break;
+                    }
+                    case DataPlant::LoopDemandCalcScheme::DualSetPointDeadBand: {
+                        if (state.dataPlnt->PlantLoop(LoopNum).OpScheme(SchemeNum).EquipList(1).Comp(CompNum).CtrlType == CtrlType::CoolingOp) {
+                            if (state.dataLoopNodes
+                                    ->Node(state.dataPlnt->PlantLoop(LoopNum).OpScheme(SchemeNum).EquipList(1).Comp(CompNum).SetPointNodeNum)
+                                    .TempSetPointHi == SensedNodeFlagValue) {
+                                if (!state.dataGlobal->AnyEnergyManagementSystemInModel) {
+                                    ShowSevereError(state,
+                                                    "Missing temperature high setpoint for " + CurrentModuleObject + " named " +
+                                                        state.dataIPShortCut->cAlphaArgs(1));
+                                    ShowContinueError(
+                                        state,
+                                        "A temperature high setpoint is needed at the node named " +
+                                            state.dataPlnt->PlantLoop(LoopNum).OpScheme(SchemeNum).EquipList(1).Comp(CompNum).SetPointNodeName);
+                                    if (state.dataPlnt->PlantLoop(LoopNum).TypeOfLoop == LoopType::Plant) {
+                                        ShowContinueError(state,
+                                                          "PlantLoop=\"" + state.dataPlnt->PlantLoop(LoopNum).Name +
+                                                              "\", Plant Loop Demand Calculation Scheme=DualSetpointDeadband");
+                                    } else if (state.dataPlnt->PlantLoop(LoopNum).TypeOfLoop ==
+                                               LoopType::Condenser) { // not applicable to Condenser loops
+                                    }
+                                    ShowContinueError(state, " Use a setpoint manager to place a dual temperature setpoint on the node");
                                     ErrorsFound = true;
                                 } else {
                                     // need call to EMS to check node
                                     NodeEMSSetPointMissing = false;
                                     CheckIfNodeSetPointManagedByEMS(
                                         state,
                                         state.dataPlnt->PlantLoop(LoopNum).OpScheme(SchemeNum).EquipList(1).Comp(CompNum).SetPointNodeNum,
-                                        EMSManager::SPControlType::iTemperatureSetPoint,
+                                        EMSManager::SPControlType::iTemperatureMaxSetPoint,
```

Comment:
Unexpected renaming, I'll have to figure out why that is needed.

Reply:
I think this is GitHub getting confused about what to compare against? I think it should be comparing against line 1601 in Develop. Instead it is comparing against line 1550. It seems to be showing up ok in the CLion diff viewer. 
![image](https://user-images.githubusercontent.com/45446967/142028378-2f6cc48b-8f77-4b2e-82e4-366bcb623314.png)